R/methods.R

```diff
@@ -98,7 +99,9 @@ setMethod("join_features", "Seurat",  function(.data,
                                                features = NULL,
                                                all = FALSE,
                                                exclude_zeros = FALSE,
-                                               shape = "long", ...)
+                                               shape = "long",
+                                               assay = .data@active.assay, 
```

Comment:
OK I see now that we can find a way in between.

1) `join_feature` can have assay = NULL by default

2) `get_abundance_sc_wide` can have `assay = Assays(pbmc_small)` by default, and inside the function you can use `assay` instead of `assay_names`.

3) `get_abundance_sc_long` can have assay = .data@active.assay by default

Reply:
Thank you very much for your guidance. I set `join_feature` to NULL by default, and set default values for `get_abundance_sc_long` and `get_abundance_sc_wide`. I apologize if I am missing something.